cli/src/cli_builder.rs

```diff
@@ -2,21 +2,23 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-// Copyright 2023 Oxide Computer Company
+// Copyright 2023-2024 Oxide Computer Company
```

Comment:
Now I feel like a real jerk, but I think we just put the most recent year in our copyright notices. 

```suggestion
// Copyright 2024 Oxide Computer Company
```

https://github.com/oxidecomputer/meta/blob/b71cf2cde756ec8012eeb6822bac470b9ed5ba7e/engineering/code.md?plain=1#L5

Reply:
I'm blaming my choice to do this on a previous job with over zealous lawyers.  I don't actually care as long as I don't have to sit through a yearly training explaining the importance of copyright year updates.

Reply:
Get 4 lawyers in a room; get 5 answers